browser_utils.js

```diff
@@ -241,18 +273,26 @@ async function waitForText(page, text, {timeout=30000, extraMessage=undefined}={
     }
 }
 
+/**
+ * @hidden
+ */
 function _checkTestId(testId) {
     if (typeof testId !== 'string') throw new Error(`Invalid testId type ${testId}`);
     assert(/^[-a-zA-Z0-9_.]+$/.test(testId), `Invalid testId ${JSON.stringify(testId)}`);
 }
 
 /**
- * @param {import('puppeteer').Page} page 
- * @param {string} testId 
- * @param {{extraMessage?: string, timeout?: number, visible?: boolean}} [options] 
- * @returns {Promise<import('puppeteer').ElementHandle>}
+ * Search for an element with the given `data-testid` attribute.
+ *
+ * @param {import('puppeteer').Page} page puppeteer page object.
+ * @param {string} testId The test id to search
+ * @param {{extraMessage?: string, timeout?: number, visible?: boolean}} [__namedParameters] Options (currently not visible in output due to typedoc bug)
+ * @param {string?} extraMessage Error message shown if the element is not visible in time.
+ * @param {number?} timeout How long to wait, in milliseconds.
+ * @param {boolean?} visible Whether the element must be visible within the timeout.
```

Comment:
should mention the default (`default: true`)

Reply:
Fixed.